### PR TITLE
Complete scene 5

### PR DESCRIPTION
**Pull request by ahmedhamodi**

### CODE REVIEW

File: scripts/mockdata/scenes_e1s5.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Bedroom with Cat",
```
Comment (ahmedhamodi) [inline scripts/mockdata/scenes_e1s5.json:3]:
```suggestion
    "name": "Bedroom with Cat - 3",
```

File: scripts/mockdata/scenarios_e1.json [hunk 2]
```diff
@@ -97,6 +97,28 @@
           }
```
Comment (drbalar) [inline scripts/mockdata/scenarios_e1.json:106]: I think we're missing a few slides for the scene intro here 
image

**Review by drbalar — APPROVED** [review]
Looks good 🚀

File: scripts/mockdata/scenarios_e1.json (hunk 2 above)
Comment (ahmedhamodi) [reply scripts/mockdata/scenarios_e1.json:106]: Added 👌

**Pull request merged**